WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1354,6 +1355,21 @@ private void showPublishConfirmationOrUpdateIfNotLocalDraft() {
         }
     }
 
+    private void showGutenbergInformativeDialog() {
+        // Show the GB informative dialog on editing GB posts
+        if (!mIsNewPost && !AppPrefs.isGutenbergInformativeDialogDisabled()) {
+            final PromoDialog asyncPromoDialog = new PromoDialog();
```

Comment:
let's rename `final PromoDialog asyncPromoDialog` to `final PromoDialog gbInformativeDialog`